Formula/clisp.rb

```diff
@@ -38,7 +38,7 @@ class Clisp < Formula
 
   def install
     ENV.deparallelize # This build isn't parallel safe.
-    ENV.O0 # Any optimization breaks the build
+    ENV["HOMEBREW_OPTIMIZATION_LEVEL"] = "O0" # Any optimization breaks the build
```

Comment:
Let's never do this, it's a private API/implementation detail 😁

Reply:
That's why I hadn't merged it yet. 🙂